app/views/admin/invoices/show.html.erb

```diff
@@ -1 +1,13 @@
-<h1><%= @invoice.id %></h1>
+<h1> Id: <%= @invoice.id %></h1>
+<p> Status: <%= @invoice.status %></p>
+<p> Creation Date: <%= @invoice.creation_date_formatted %></p>
+<p> Customer Name: <%= @customer.first_name %> <%= @customer.last_name %></p>
+
+<% @items.each do |item| %>
+   <% item.invoice_items.each do |invoice_item| %>
+    <h3>Item Name: <%= item.name %></h3>
```

Comment:
I think this each block will have the same flawed logic I had earlier and display invoice_items that are not associated with this invoice. If that is the case I have written a method that should be able to fix it, but we can check on heroku once this is merged to main.